hieradata/dev/wso2/common.yaml

```diff
@@ -51,10 +51,14 @@ wso2::template_list:
   - bin/wso2server.sh
 #  - bin/ciphertool.sh
 
-# Puppet file list to be copied
+# Puppet file list to be copied triggering restart
```

Comment:
I think these are not puppet files. They can be any files used in the product. Moreover, it might be better to make this sentence more clear, shall we rephrase this to something similar to "File list to be copied to the product folder, if there are changes found the system service will be restarted".